src/home/home.scss

```diff
@@ -51,25 +51,29 @@
         justify-content: space-between;
         cursor: pointer;
         width: 710px;
-
+        gap: 24px;
```

Comment:
We can't use "gap" or "row-gap" or "column-gap" with flex.  (Doesn't work reliably with iOS).  But we can use them with grid, which is why our best practice, except in trivial cases, is to use grid, not flex.

But it looks great to me.  Switch to `grid` and this should be acceptable.